tests/test_serialization.py

```diff
@@ -0,0 +1,42 @@
+import pickle
+
+from pytest import mark
+
+from vantage6.tools import serialization
+import pandas as pd
+
+JSON = 'json'
+
+
+@mark.parametrize("data,target", [
+    # Default serialization
+    ([1, 2, 3], '[1, 2, 3]'),
+    ('hello', '"hello"'),
+    ({'hello': 'goodbye'}, '{"hello": "goodbye"}'),
+
+    # Pandas serialization
+    (pd.DataFrame([[1, 2, 3]], columns=['one', 'two', 'three']), '{"one":{"0":1},"two":{"0":2},"three":{"0":3}}'),
+    (pd.Series([1, 2, 3]), '{"0":1,"1":2,"2":3}')
+])
+def test_json_serialization(data, target):
```

Comment:
It feels a bit like you are actually testing `pandas` and `jsonpickle` code instead of your own code, but I guess testing only your own code would make your tests maybe more complicated

Reply:
Hm yeah I get your point but I don't really see a reasonable way to factor out pandas and the json package

Reply:
Fair enough